DESCRIPTION

```diff
@@ -50,7 +50,8 @@ Imports:
     styler (>= 1.8.1),
     Hmisc,
     MASS,
-    boot
+    boot,
+    tidyselect
```

Comment:
`tidyselect` mangla. Korfor har me ikkje fått feilmelding på dette tidlegare?

Reply:
Dei to aktuelle `tidyselect`-funksjonane `all_of()` og `where()` er importert i `dplyr`, og me hadde tilgang til dei ved å importerta heile `dplyr`-pakken.